lib/item.py

```diff
@@ -1,5 +1,6 @@
 import uuid
 
+
 class Inventory(object):
     def __init__(self):
         # Probably want to extend this so you can have many items of the same name
```

Comment:
probably eventually also want to have types of items, weapon, armour, potion etc.

Reply:
Yeah, I think each of those will subclass Entity and then have different stats.  You can see the start of that with the Weapon class.